src/test/java/org/springframework/samples/petclinic/PostgresIntegrationTests.java

```diff
@@ -114,7 +115,16 @@ public void printProperties() {
 				Arrays.sort(names);
 				for (String name : names) {
 					String resolved = environment.getProperty(name);
-					String value = source.getProperty(name).toString();
+
+					assertNotNull(resolved, "resolved property was expecting a string but is null.");
```

Comment:
Just in case someone messed up the properties' files. Still better than NPE :) .
fyi, I just realized that message was missing more useful details so I just modified it, will push change soon for this line.